### PR TITLE
Consistent trace vocab reference 

### PR DESCRIPTION
**Pull request by nissimsan**

### CODE REVIEW

**Review by brownoxford — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]

File: docs/spec/index.html [hunk 1]
```diff
@@ -272,8 +276,7 @@ <h3>CCG Standards Stack</h3>
         <li>
           <em>Linked Data</em> establishes a concise semantic context of the
           data exchanged, enabled with JSON-LD by
-          the <a href="https://w3id.org/traceability">Traceability
-            Vocabulary</a>.
+          the [[[TRACE-VOCAB]]].
```
Comment (TallTed) [inline docs/spec/index.html:279]:
```suggestion
          the [[[TRACE-VOCAB]]] [[TRACE-VOCAB]].
```

File: docs/spec/index.html [hunk 2]
```diff
@@ -299,8 +302,7 @@ <h3>CCG Standards Stack</h3>
     <h2>Terminology</h2>
     <p>
       Term definitions for this specification can be found in the
-      <a href="https://w3id.org/traceability/#terminology">
-        Traceability Vocabulary </a>
+      [[[TRACE-VOCAB]]]
       document. This specification does not define any additional terms.
```
Comment (TallTed) [inline docs/spec/index.html:306]:
```suggestion
      [[[TRACE-VOCAB]]] [[TRACE-VOCAB]].
      This specification does not define any additional terms.
```

File: docs/spec/sections/abstract.html [hunk 3]
```diff
@@ -4,8 +4,8 @@ <h2>Abstract</h2>
   <p>
     This specification describes discovery and exchange mechanisms of [Verifiable
     Credentials](https://www.w3.org/TR/vc-data-model-2.0/), particularly as they relate to supply chain use cases.
-    Exchanged Verifiable Credentials often implement schemas defined in the [Traceability Vocab
-    Specification](https://w3c-ccg.github.io/traceability-vocab/).
+    Exchanged Verifiable Credentials often implement schemas defined in the Traceability Vocabulary
+    Specification [[TRACE-VOCAB]].
```
Comment (TallTed) [inline docs/spec/sections/abstract.html:8]:
```suggestion
    Exchanged Verifiable Credentials often implement schemas defined in
    the [[[TRACE-VOCAB]]] [[TRACE-VOCAB]]..
```

**Review by TallTed — CHANGES_REQUESTED** [review]
ReSpec magic

**Review by brentzundel — APPROVED** [review]

**Comment by mkhraisha** [comment]
Please fix the change to the vc data model reference previously merged in and merge out of band

**Comment by nissimsan** [comment]
Fixed the overwrite from merge conflict. Merging.

**Pull request merged**